src/main/java/org/broadinstitute/hellbender/tools/exome/CnvCaller.java

```diff
@@ -27,6 +28,8 @@
  *
  * Note:  Assumes that the incoming target file is in log_2(CR)
  * Note:  Assumes that the incoming segment means are CR.
+ * Note: Assumes that the input @{link ReadCountCollection} contains only one sample -- calls will be made for only one
```

Comment:
hmm, i'm still not quite sure about this---why do we need any of the tools to work with multiple samples (beside the obvious ones that deal with the PoN that require it or may see a performance boost)?

for example, what is gained by allowing multiple samples to be passed to the caller in a single file/object if they are each going to be called individually anyway?  shouldn't we just rely on the workflow scripts to take care of this?  especially considering that the natural output of nearly all of the steps will be a single file for each sample---e.g., plotting and segmentation (because of our reliance on the R script)---i'm not sure what is gained by artificially aggregating the inputs into a single file.  however, maybe i'm missing something about how multiple samples were treated in recapseg/previous iterations?